### ISSUE TITLE
NPM: not detecting postinstall scripts

### ISSUE THREAD

**Issue by emilva** [DataDog/guarddog#175]
Hello team! Very cool tool you have created, and I am glad for the NPM support!

When testing this tool against different malicious packages I get no results, I took som examples from https://github.com/spaceraccoon/npm-zoo

For example:
https://github.com/spaceraccoon/npm-zoo/tree/master/packages/pizza-pasta/pizza-pasta-1.0.3
```
guarddog npm scan package.json
Found 0 potentially malicious indicators scanning package.json
```

```
guarddog npm verify package.json
Scanning using at most 16 parallel worker threads
```

Should this one be detected, or am not running the correct commands?

Cheers,

emil

**Comment by christophetd** [comment]
Thanks for the report! We'll look into it shortly and get back to you

**Comment by emilva** [comment]
Thanks @christophetd that clear things up!

I did try with `guarddog npm scan .` to scan the local directory, but then it also tried to check against the registry. 

Cheers,

emil

**Comment by christophetd** [comment]
that's a good point, it should consider it a local path

**Comment by christophetd** [comment]
Should be addressed by #176. Thanks for the report!

**Comment by emilva** [comment]
Nice, you are too fast! Thanks!
